packages/realm-web/src/environment.ts

```diff
@@ -0,0 +1,80 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+import { Storage } from "./storage";
+import { Window } from "./OAuth2Helper";
+
+/**
+ * A simplified interface for the console.
+ * This should reflect the lowest common denominator across environemnts.
+ */
+interface Console {
+    /**
+     *
+     */
```

Comment:
???

Reply:
I updated these, but also moved them out of the environment and onto `environment.d.ts` since they are available as globals, it seems strange to include them in the environment object like this.